### PR TITLE
add  echooowallet

### PR DESCRIPTION
**Pull request by rqzrqh**
### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [x] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

#### Tests with demo app (SDK)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

### CODE REVIEW

**Review by Adamj1232 — CHANGES_REQUESTED** [review]
Hi @rqzrqh thank you for the PR to support echooo wallet. 
As this is an injected wallet we do not support adding a new module specific for the injected wallet as module upkeep is extensive and all actions of adding a whole new module for an injected wallet can be done through the injected-module package.
If you can remove the echoo package, echoo page within the docs and CI pipeline changes we should be gtg.

**Comment by lianshao123** [comment]
> 你好@rqzrqh感谢您的PR支持echooo钱包。 由于这是一个注入钱包，我们不支持添加特定于注入钱包的新模块，因为模块维护非常广泛，并且为注入钱包添加全新模块的所有操作都可以通过注入模块包完成。 如果你可以删除 echoo 包，文档中的 echoo 页面和 CI 管道更改我们应该是 gtg。

**Comment by lianshao123** [comment]
Is there any documentation that can help me complete the import of wallet plugin to blocknative? @Adamj1232

**Comment by Adamj1232** [comment]
@lianshao123 Steps to get this PR ready to go would be:

1. Remove packages/echooo
2. revert echooo package changes to .circleci/config.yml
3. remove the [...36]echooo/+page.md
4. remove web3-onboard/echo from docs/package.json & services/onboard.js

Here is some documentation around adding a wallet and contributing to the project - https://onboard.blocknative.com/docs/overview/contribution-guide#how-can-i-add-a-new-wallet-%F0%9F%92%B3

**Comment by lianshao123** [comment]
One version has been submitted, please review it @Adamj1232

File: packages/demo/package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
     "@web3-onboard/xdefi": "^2.0.5",
     "@web3-onboard/zeal": "^2.0.3",
     "@web3-onboard/particle-network": "^2.0.1",
+    "@web3-onboard/echooo": "2.0.1-alpha.1",
```
Comment (Adamj1232) [inline packages/demo/package.json:67]:
```suggestion
```

**Review by Adamj1232 — APPROVED** [review]
Looks good! Thank you for contributing to the project!

**Pull request merged**